exercises/03-xpath-intro.md

```diff
@@ -9,47 +9,50 @@
 
 Ideally, web development and test automation goes hand in hand, so
 that the people who develop the websites, would also do (at least
-a bit of) test automation. However, more often than not this is not
-the case and that causes trouble to test automation developers.
-Websites are usually creates "UX first", which is good for the users,
+a bit of) test automation, or be aware of it. However, more often than not this is not
+the case and that causes trouble for test automation developers.
+Websites are often created with "UX first" approach, which is good for the users,
 but it typically means more work for the test automation developers.
```

Comment:
I feel this is controversial statement. In my view, messy UI code is precisely caused by *not* creating the application with "UX first" approach. 

Usually when things are designed and thought through before hand, the frontend code is clean with good element id's & classes and thus, easily testable. It is precisely people hacking together some crap that results in spaghetti javascript, invalid HTML, weird `<iframe>`'s and the whole shebang. 

This is also the reason why it would be good idea for the developers to write test automation themselves -- because then they would have to "eat their own dog food" ie. deal with the spaghetti they have created.

Reply:
That is a good point. The paragraph could be rephrased to something like

> Unfortunately, websites are often created with little emphasis on automation, which means features work perfectly fine for a human user, but are very tedious to automate.